components/sections/home-page/Newsletter.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import SectionWrapper from '../../common/layout/SectionWrapper'
+import { Heading, Typography } from '../../common/text'
+import GradientBorderWrapper from '../../common/GradientBorderWrapper'
+import Image from 'next/image'
+
+import StrokeR3 from '../../../public/_3_rightStroke.svg'
+import StrokeL3 from '../../../public/_3_leftStroke.svg'
+import StrokeMobile2 from '../../../public/background-strokes/stroke_mobile_2.svg'
+import StrokeMobile3 from '../../../public/background-strokes/stroke_mobile_3.svg'
+
+const Newsletter = () => {
+  return (
+    <SectionWrapper pt={100} pb={250}>
+        <div className="relative w-screen">
+          <div className="absolute largeTablet:block top-[-14rem] largeTablet:-top-[10rem] left-[-30rem] largeTablet:left-[-15rem]">
+            <Image alt="Doodles" src={StrokeL3} />
+          </div>
+
+          <div className="absolute largeTablet:block top-[-14rem] largeTablet:-top-[22rem] right-[-34rem] largeTablet:-right-[32rem]">
+            <Image alt="Doodles" src={StrokeR3} />
+          </div>
+
+          <div className="absolute bottom-[51rem] right-0  largeTablet:hidden">
+            <Image alt="Doodles" src={StrokeMobile2} />
+          </div>
+
+          <div className="absolute bottom-[32rem] left-0  largeTablet:hidden">
+            <Image alt="Doodles" src={StrokeMobile3} />
+          </div>
+        <div className="flex flex-col items-center gap-y-8">
+          
+        <Heading alignSmall="center">
+          $orange-to-yellowThe fast track$orange-to-yellow to open source
+        </Heading>
```

Comment:
There are two newsletter sections and should be one. We only need email, no name as well.

<img width="881" alt="Screen Shot 2023-04-26 at 5 33 13 AM" src="https://user-images.githubusercontent.com/5713670/234576618-88c29881-6d7b-4ed1-af98-dde0355e0d14.png">

<img width="692" alt="Screen Shot 2023-04-26 at 5 32 54 AM" src="https://user-images.githubusercontent.com/5713670/234576661-17faa5ef-f15f-4685-a751-7f5d9e25357e.png">

Reply:
Hey @shamimbinnur, let's delete the "Subscribe for Extra Sauce" newsletter section and keep the "The fast track..." section as is (without the 'Your Name' of course).  Could we also switch out the copy from "Sign Up" to "Subscribe"? 

Thanks